src/main/java/com/minecolonies/rankup/modules/core/config/GroupsConfig.java

```diff
@@ -29,8 +36,8 @@
         @Setting(value = "group-rank", comment = "The rank of which the group is, rank one is a starting rank of a player, rank 2 being the next rank they will gain. Set to -1 to ignore")
         public int rank = 0;
 
-        @Setting(value = "group-commands-on-rankup", comment = "When someone ranks up into this group all these commands are run. \n you may use {player} and {group} attributes")
-        public List<String> commands = Arrays.asList("say {player} joined group {group}", "give {player} minecraft:dirt");
+        @Setting(value = "group-commands-on-rankup", comment = "When someone ranks up into this group all these commands are run. \n you may use " + Constants.PlayerInfo.PLAYER_NAME + " and " + Constants.PlayerInfo.PLAYER_GROUP + " attributes")
+        public List<String> commands = Arrays.asList("say " + Constants.PlayerInfo.PLAYER_NAME + " joined group " + Constants.PlayerInfo.PLAYER_GROUP + "", "give " + Constants.PlayerInfo.PLAYER_NAME + " minecraft:dirt");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 220 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS00103)